listener.js

```diff
@@ -179,11 +185,35 @@ server.post("/", (payload, res) => {
             // CHANGE DEVICE BRIGHTNESS
             case "brightness":
                 if (device.name == target.device) {
-                    let brightness = await cli_exec("curl -X POST http://" + device.ipaddr + ":8080/brightness?value=" + target.value, "device_command");
+                    var ipaddr = '';
+                    if (config.manual_ip) {
+                        ipaddr = device.ipaddr;
+                    }
+                    else {
+                        // Look for WiFi addresses since it's quick
+                        ipaddr = await cli_exec("ping -t 1 " + device.name, 'device_ipaddr');
+                        if (ipaddr == '') {
```

Comment:
```suggestion
                        if (!ipaddr) {
```

Reply:
I updated the script with the latest commit. I'll mark these as resolved.